README.md

```diff
@@ -1,4 +1,4 @@
-# Getting Started with Create React App - Math Magicians
+# Getting Started with Create React App - Math Magicians App
 
 This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
 
```

Comment:
**Your project running smoothly on a browser, it's perfect! 👏**

However, there is a small gap that needs to be filled in before approving this PR. 

- [ ] Please update your README.MD file as this given [template](https://github.com/microverseinc/readme-template/blob/master/README.md). Last but not least, do not forget to add your MIT License to it.